src/index.ts

```diff
@@ -1,5 +1,6 @@
 import dispatcher from './dispatcher';
 
+export { DispatcherContext, createDispatcher } from './dispatcher';
```

Comment:
Nit: should we just export a Provider here?

Reply:
Fixed in 966f4da. LMK if you think there would be a better name for the Provider.